.github/workflows/github-actions.yml

```
@@ -0,0 +1,41 @@
+name: GitHub Actions
+on: [push]
```

```suggestion
on: 
  pull_request:
```

This workflow should probably only be triggered when PRs are created or updated.